cpp/benchmarks/io/parquet/parquet_writer_chunks.cpp

```diff
@@ -94,11 +114,18 @@ void PQ_write_chunked(nvbench::state& state)
       encoded_file_size = source_sink.size();
     });
 
-  auto const time = state.get_summary("nv/cold/time/gpu/mean").get_float64("value");
-  state.add_element_count(static_cast<double>(data_size) / time, "bytes_per_second");
+  auto const time       = state.get_summary("nv/cold/time/gpu/mean").get_float64("value");
+  auto const throughput = static_cast<double>(data_size) / time;
+
+  state.add_element_count(throughput, "bytes_per_second");
   state.add_buffer_size(
     mem_stats_logger.peak_memory_usage(), "peak_memory_usage", "peak_memory_usage");
   state.add_buffer_size(encoded_file_size, "encoded_file_size", "encoded_file_size");
+  state.add_buffer_size(chunk_size_bytes, "chunk_size", "chunk_size_bytes");
```

Comment:
not sure if this is useful, we have the data size and the number of chunks already